samples/cpp/benchmark_app/main.cpp

```diff
@@ -490,7 +490,12 @@ int main(int argc, char* argv[]) {
             // --------------------------------------------------------
             next_step();
             startTime = Time::now();
+            std::map<std::string, Parameter> options = {{"MODEL_PTR", cnnNetwork.getFunction()}};
+            auto max_batch_size = ie.GetMetric("GPU.1", GPU_METRIC_KEY(MAX_BATCH_SIZE), options).as<unsigned int>();
+            std::cout << "max batch size is " << max_batch_size << std::endl;
```

Comment:
Should be removed, shouldn't it?

Reply:
ah this is added by mistake...will update.